debug/index.js

```diff
@@ -27,7 +27,7 @@ mapDiv.style.bottom = 0;
 
 var map = new mapboxgl.Map({
   container: mapDiv,
-  style: 'mapbox://styles/mapbox/streets-v9',
+  style: 'mapbox://styles/mapbox/streets-v11',
```

Comment:
It should be either `streets-v12`

```suggestion
  style: 'mapbox://styles/mapbox/streets-v12',
```

…or Standard

```suggestion
  style: 'mapbox://styles/mapbox/standard',
```

Reply:
Updated to v12. Thank you!